### PR TITLE
add connection string to mongo connection

### LINKED ISSUE #79: Configurable multiple mongo hosts with connection string
YOU CAN'T CONNECT TO MORE THAN A SINGLE MONGODB SERVER, WTF.
```
const mongoHost = process.env.MONGO_HOST || config.db.host;
    await mongoose.connect(
      `mongodb://${mongoHost}:${config.db.port}/${config.db.name}`,
      { useCreateIndex: true, useNewUrlParser: true, useFindAndModify: false }
    );
```

### PR DESCRIPTION
**Pull request by Shahar-Y**
fixes #79, #80 
also, fixes my linter problems
Signed-off-by: shahar-Y <shya95@gmail.com>

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -65,3 +65,8 @@ function getConfig(confType: string) : Config {
 }
 
 export const config : Config = getConfig(process.env.NODE_ENV || dev.conf_type);
+const mongoHost = process.env.MONGO_HOST || config.db.host;
```
Comment (omrishtam) [inline src/config.ts]: Remove `mongoHost` OR `connectionString` and export `process.env.MONGO_HOST || 'mongodb://localhost:27017';`.

**Comment by omrishtam** [comment]
https://docs.mongodb.com/manual/reference/connection-string/

**Comment by Shahar-Y** [comment]
now it's not signed and I have to force it :(

**Comment by omrishtam** [comment]
> now it's not signed and I have to force it :(

Sorry I tried to help @omerzamir from my pc at home and I don’t have my gpg key there.

**Pull request merged**